### PR TITLE
created feedback file in  /src/feedback- issue #12

### PR DESCRIPTION
**Pull request by amanuely**
@dogi @paulbert  Please have a look at PR

### CODE REVIEW

File: src/app/home/home.module.ts [hunk 1]
```diff
@@ -25,9 +26,12 @@ import { NationComponent } from '../nation/nation.component';
     NavigationComponent,
     UsersComponent,
     CoursesComponent,
+    FormErrorMessagesComponent,
```
Comment (lmmrssa) [inline src/app/home/home.module.ts]: this is duplicated

File: src/app/feedback/feedback.component.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+			</div>
+			<div *ngIf="! isFeedbackMsg" class="alert alert-danger">{{  msgForUsr}}</div>
+		</div>
+	</div>
+	<div>
+		<br>
+		<button  type="submit" class="btn btn-success">Submit</button>
+	</div>	
+</form>
+
```
Comment (lmmrssa) [inline src/app/feedback/feedback.component.html]: remove extra spaces

File: src/app/feedback/feedback.component.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+			<div class="form-group col-2">
+				<label> Feedback </label>
+			</div>
+			<div class="form-group col-10">
+				<textarea type="text" name=" feedbackMsg" [(ngModel)]="feedback.feedbackMsg"	[rows]="6" [cols]="40"  required="true"></textarea>
+			</div>
+			<div *ngIf="! isFeedbackMsg" class="alert alert-danger">{{  msgForUsr}}</div>
+		</div>
+	</div>
+	<div>
```
Comment (lmmrssa) [inline src/app/feedback/feedback.component.html]: need to fix intendation

**Comment by lmmrssa** [comment]
need to remove empty scss file and add database script to create table

File: src/app/home/home-router.module.ts [hunk 4]
```diff
@@ -5,8 +5,12 @@ import { DashboardComponent } from '../dashboard/dashboard.component';
 import { UsersComponent } from '../users/users.component';
 import { HomeComponent } from './home.component';
 import { CoursesComponent } from '../courses/courses.component';
+
```
Comment (lmmrssa) [inline src/app/home/home-router.module.ts]: empty space

File: src/app/feedback/feedback.component.html (hunk 2 above)
Comment (razu9861) [inline src/app/feedback/feedback.component.html]: extra lines need to be removed

File: src/app/feedback/feedback.component.ts [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  message:string;
+  fedbkSubmitted:boolean= false;
+
+  constructor(
+    private userService: UserService,
+    private couchService:CouchService
+   ) { }
+
+  ngOnInit() {
+    this.feedback.name= this.userService.get().name;e
```
Comment (lmmrssa) [inline src/app/feedback/feedback.component.ts]: i think it needs to remove last e

**Comment by lmmrssa** [comment]
could you rebase this branch again to reflect new changes on form.
Also it needs to have "i18n" attributes on the template before all static text.

File: src/app/feedback/feedback.component.ts [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+      .then((data) => {
+        this. msgForUsr='';
+        this.isFeedbackMsg=false;
+        this.fedbkSubmitted=true;
+        this. msgForUsr= "Thank you! We have received your feedback";
+      },
+      (error) => {
+        this.isFeedbackMsg=true;
+        this.fedbkSubmitted=true;
+        this.msgForUsr = 'Error with submitting your feedback';
```
Comment (lmmrssa) [inline src/app/feedback/feedback.component.ts]: try to have the same quote either or single or double based on style guide

**Comment by paulbert** [comment]
Also @dogi @lmmrssa this is another instance where the BeLL database structure is a little different.  It looks like feedback goes into a `report` database with fields as pictured below.  Should we work off of that or is this something where we can diverge from the old application and create a new data structure?

feedback example

**Comment by mappuji** [comment]
Do review with this step please 
https://www.youtube.com/watch?v=SmZgAULwoh4

**Comment by paulbert** [comment]
Closing this since it is a duplicate of #173

**Pull request closed without merging**